src/minio.d.ts

```diff
@@ -207,6 +207,9 @@ export class Client extends TypedClient {
   removeBucketTagging(bucketName: string, callback: NoResultCallback): void
   removeBucketTagging(bucketName: string): Promise<void>
 
+  setBucketVersioning(bucketName: string, versioningConfig: any, callback: NoResultCallback): void
```

Comment:
why add these two methods?

Reply:
This would be cleaned up in a new PR as we decided to drop callback support midway.